src/service/fixed-layer.js

```diff
@@ -162,6 +234,19 @@ export class FixedLayer {
           ' as Google\'s because the fixed or sticky positioning might have' +
           ' slightly different layout.');
     }
+  }
+
+  /**
+   * @param {!Node} node
+   * @param {boolean=} opt_lightboxMode
+   * @private
+   */
+  scanNode_(node, opt_lightboxMode) {
+    this.trySetupSelectorsNoInline(node, devAssert(this.fixedSelectors_),
+        devAssert(this.stickySelectors_), opt_lightboxMode);
+
+    // Sort tracked elements in document order.
+    this.sortInDomOrder_();
```

Comment:
I never understood why this was necessary...

Reply:
Avoid order change for overlapping fixed elements due to z-index change?